static/templates/settings/image_upload_widget.hbs

```diff
@@ -1,4 +1,10 @@
-<div id ="{{widget}}-upload-widget" class="inline-block image_upload_widget">
+<div id="{{widget}}-upload-widget" class="inline-block image_upload_widget">
+    <div class="image--disabled {{#if is_editable_by_current_user}}hide{{/if}}">
+        <div class="image-upload-background"></div>
+        <span class="image--disabled__text flex" aria-label="{{ disabled_text }}" role="button" tabindex="0">
```

Comment:
We should call this `image-disabled-text` (and `image-disabled` above) for consistency.

Reply:
fixed, thank you.